admin/controllers/school.js

```diff
@@ -368,18 +372,83 @@ const getSelectPupilNotTakingCheck = async (req, res, next) => {
 
   let attendanceCodes
   let formData
+  let pupilsList
+  let htmlSortDirection = []
+  let arrowSortDirection = []
 
+  // Sorting
+  const sortField = req.params.sortField === undefined ? 'name' : req.params.sortField
```

Comment:
Don't we want the sorting to be extracted to a service as well?
Even though we are skipping tests for now the service refactoring should happen as part of this ticket I believe.

Reply:
I saw you had a comment about it on the other PR.
Does this apply to this one as well?

Reply:
@george yes, as the PR description says i'd like to decouple the controller into at least two services, that comes next. Plus, prior to create a service to deal sorting with sorting I want a chat with UX to make sure I'll cover any possible future scenario too.